src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -34,6 +34,12 @@
 #include "serviceRoutines/postUpdateContext.h"
 
 
+static bool legalEntityLength(Entity* eP, const std::string &servicePath)
+{
+  const int constant = 10;
```

Comment:
I'd suggest to use a #define in the file preamble (i.e. just after the #include lines):

```
#define STRUCTURAL_OVERHEAD_BSON_ID 10
```

(The name is to align with the terms used by MongoDB documentation... "contant" is too broad)